session/bootstrap.go

```diff
@@ -619,11 +619,13 @@ const (
 	version90 = 90
 	// version91 converts prepared-plan-cache to sysvars
 	version91 = 91
+	// version92 converts performance.force-priority to sysvars
+	version92 = 92
```

Comment:
Bootstrap `importConfigOption` is for importing config itmes into sysvars. The function writes the last config value into the sysvars table. Since these are staying config for now, you shouldn't need a bootstrap task.